astroquery/simbad/core.py

```diff
@@ -382,15 +383,17 @@ def add_votable_fields(self, *args):
         # the legacy way of adding fluxes
         args = list(args)
         fluxes_to_add = []
-        for arg in args:
+        args_copy = args.copy()
+        for arg in args_copy:
             if arg.startswith("flux_"):
                 raise ValueError("The votable fields 'flux_***(filtername)' are removed and replaced "
                                  "by 'flux' that will add all information for every filters. "
                                  "See section on filters in "
                                  "https://astroquery.readthedocs.io/en/latest/simbad/simbad_evolution.html"
                                  " to see the new ways to interact with SIMBAD's fluxes.")
             if re.match(r"^flux.*\(.+\)$", arg):
-                warnings.warn("The notation 'flux(U)' is deprecated since 0.4.8 in favor of 'U'. "
+                warnings.warn("The notation 'flux(XXX)' is deprecated since 0.4.8 in favor of 'XXX'. "
```

Comment:
Could we use XYZ or some other placeholder here instead?

Reply:
even better would be to put in the one being matched, but I suppose it's not worth the effort of patching it out from the regex.

Reply:
Done :slightly_smiling_face: 
(It was just me being lazy)